src/routes/trip.route.js

```diff
@@ -3,6 +3,11 @@ import { tripController } from '../controllers';
 import { validate, errorHandler, isLoggedIn } from '../middlewares';
 import * as auth from '../middlewares/authorize';
 import { tripSchema, approveRequestSchema } from '../validations';
+import {
+  staticsValidate,
+  periodValidate
+} from '../validations/statisticsValidation';
```

Comment:
Export these files from validation index and import them together with others from validations directory